config/appConfig.js

```diff
@@ -13,11 +17,14 @@ export default {
     tokenUrl: process.env.OAUTH_TOKEN_URL || 'https://isso.nypl.org/oauth/token',
     loginUrl: process.env.OAUTH_LOGIN_URL || 'https://isso.nypl.org/auth/login',
     clientId: process.env.CLIENT_ID || 'platform_admin',
-    // clientSecret should be put in .env file
+    clientSecret: process.env.CLIENT_SECRET,
     callbackUrl: process.env.NODE_ENV === 'production' ? process.env.OAUTH_CALLBACK_URL : 'http://local.nypl.org:3001/callback',
-    tokenUrlForNyplApiClient: 'https://isso.nypl.org/',
+  },
+  nyplMicroService: {
+    tokenUrlForNyplApiClient: process.env.TOKEN_URL_FOR_NYPL_API_CLIENT,
+    platformBaseUrl: process.env.PLATFORM_BASE_URL,
     refileRequestId: process.env.REFILE_REQUEST_ID || 'refile_request_service',
-    // refileRequestSecret should be put in .env file
+    refileRequestSecret: process.env.REFILE_REQUEST_SECRET,
```

Comment:
And here — I'd remove this from the config file. Otherwise people might get the idea they should put secrets in `appConfig`